packages/less/README.md

```
@@ -1,13 +1,78 @@
-# [Less.js](http://lesscss.org)
+<p align="center"><img src="http://lesscss.org/public/img/less_logo.png" width="264" height="117"></p>
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)